daemon/daemon.go

```diff
@@ -69,6 +69,7 @@ import (
 	"github.com/cilium/cilium/pkg/workloads"
 	"github.com/cilium/cilium/pkg/workloads/containerd"
 
+	"github.com/cilium/cilium/pkg/maps/datametrics"
```

Comment:
Real minor nit, but why "datametrics", and not just "metrics"? It's already clear that it's for the datapath because it's a map (or the underlying code is in the datapath).

Reply:
Because I wanted to make this a package and we already have `metrics` package in `pkg/metrics`. I initially thought that since this is under `maps`, defining  a package named `metrics` in `pkg/maps ` will still work but I get errors in that case, when I try to use the `pkg/metrics` somewhere. Hence I named it to `datametrics` to keep things simple. Please let me know if there is a golang style I need to be aware of here?

Reply:
I did not want to combine the `pkg/metrics` with this one, because `pkg/metrics` is very specific to prometheus, while this was more relevant to L3-L4 data path.

Reply:
From the golang side, most of the maps seem to be (admittedly, also redundantly...) called xxxmap, so following the style would call this metricsmap.

From the BPF datapath code, I don't think it's necessary to differentiate as this is the only metrics being performed there.

Reply:
Changed `datametrics` --> `metricsmap` on golang side.
And on BPF side, changed `data_metrics` to only `metrics`.